react/week4/user_search/src/Users.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect, useContext } from "react";
+import { UserContext } from "./UserContext";
+
+export default function Users() {
+	const { userId } = useContext(UserContext);
+	const [users, setUsers] = useState([]);
+	const [err, setErr] = useState("");
+
+	useEffect(() => {
+		if (!userId) return;
+
+		fetch(`https://api.github.com/search/users?q=${userId}`)
+			.then((res) => res.json())
+			.then((res) => {
+				setUsers(res.items);
+			})
+			.catch((err) => setErr(err.message));
+	}, [userId]);
+
+	if (!userId) return "No results";
+
+	if (err) return <p>{err}</p>;
+
+	if (userId && users.length === 0) return "Loading...";
```

Comment:
I think it should show "No results" not "Loading"... since users is an empty array, ie no users with a specific name...